src/pke/lib/scheme/ckksrns/ckksrns-fhe.cpp

```diff
@@ -2645,74 +2682,59 @@ Plaintext FHECKKSRNS::MakeAuxPlaintext(const CryptoContextImpl<DCRTPoly>& cc, co
 #endif
 
 Ciphertext<DCRTPoly> FHECKKSRNS::EvalMultExt(ConstCiphertext<DCRTPoly> ciphertext, ConstPlaintext plaintext) const {
-    Ciphertext<DCRTPoly> result = ciphertext->Clone();
-    std::vector<DCRTPoly>& cv   = result->GetElements();
-
-    DCRTPoly pt = plaintext->GetElement<DCRTPoly>();
+    auto pt = plaintext->GetElement<DCRTPoly>();
     pt.SetFormat(Format::EVALUATION);
 
-    for (auto& c : cv) {
+    auto result = ciphertext->Clone();
+    for (auto& c : result->GetElements())
         c *= pt;
-    }
     result->SetNoiseScaleDeg(result->GetNoiseScaleDeg() + plaintext->GetNoiseScaleDeg());
     result->SetScalingFactor(result->GetScalingFactor() * plaintext->GetScalingFactor());
     return result;
 }
 
 void FHECKKSRNS::EvalAddExtInPlace(Ciphertext<DCRTPoly>& ciphertext1, ConstCiphertext<DCRTPoly> ciphertext2) const {
-    std::vector<DCRTPoly>& cv1       = ciphertext1->GetElements();
-    const std::vector<DCRTPoly>& cv2 = ciphertext2->GetElements();
-    for (size_t i = 0; i < cv1.size(); ++i)
+    auto& cv1  = ciphertext1->GetElements();
+    auto& cv2  = ciphertext2->GetElements();
```

Comment:
Is it correct to drop `const` here? The intent was to say that cv2 will not be changed.